lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu
+  MENU = {'Spring Rolls' => 2.99, 'Crispy Duck' => 6.99}
+  SPACERS = 29
+
+  def raw
+    MENU
+  end
+
+  def format
+    MENU.inject("") { |result, (key, value)|
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)